plugins/org.eclipse.n4js/src/org/eclipse/n4js/postprocessing/ASTMetaInfoCache.java

```diff
@@ -109,9 +109,12 @@ public boolean hasBrokenAST() {
 			if (isCanceled()) {
 				return new Result<>(TypeRefsFactory.eINSTANCE.createUnknownTypeRef());
 			} else {
-				throw UtilN4.reportError(new IllegalStateException(
-						"cache miss: no actual type in cache for AST node: " + astNode
-								+ " in resource: " + resource.getURI()));
+				// TODO GH-73: Temporary hack. Reactivate the throw statement and solve cache miss problem, i.e. GH-95
+				Result<TypeRef> tr = new Result<>((TypeRef) astNode.eContainer());
+				return tr;
```

Comment:
but `(TypeRef) astNode.eContainer()` will throw class cast exceptions, most of the time, right?
Does #95 block #73? Then we need to fix it.